modules/migration-using-must-gather.adoc

```diff
@@ -28,6 +28,8 @@ endif::[]
 
 * You must be logged in to the {product-title} cluster as a user with the `cluster-admin` role.
 * You must have the OpenShift CLI (`oc`) installed.
+* For use with OADP 1.2 you must use {op-system-base-full} 8.x.
```

Comment:
Maybe:

* OADP 1.2: You must have the OpenShift CLI (`oc`) installed.
* OADP 1.3: You must use {op-system-base-full} {op-system-version}.

Reply:
@dfitzmau 
That isn't the intent.
The user needs to know that when using OADP 1.2 they must use RHEL8 and when using OADP 1.3 they must use RHEL9.